exe/Main.hs

```diff
@@ -84,7 +82,7 @@ main = do
 
         putStrLn "\n[3/6] Initialising IDE session"
         vfs <- makeVFSHandle
-        ide <- initialise mainRule (showEvent lock) logger (defaultIdeOptions $ return env) vfs
+        ide <- initialise mainRule (showEvent lock) logger (defaultIdeOptions $ return $ const $ return env) vfs
```

Comment:
This will need to change once multiple environments are actually used. Maybe worth adding a comment so we don’t forget about it later.